docker-compose.yml

```diff
@@ -231,20 +161,48 @@ services:
       postgres:
         condition: service_healthy
     environment:
-      - RP_DB_HOST=postgres
-      - RP_DB_USER=rpuser
-      - RP_DB_PASS=rppass
-      - RP_DB_NAME=reportportal
-      - RP_AMQP_USER=rabbitmq
-      - RP_AMQP_PASS=rabbitmq
-      - RP_AMQP_APIUSER=rabbitmq
-      - RP_AMQP_APIPASS=rabbitmq
-      - RP_BINARYSTORE_TYPE=minio
-      - RP_BINARYSTORE_MINIO_ENDPOINT=http://minio:9000
-      - RP_BINARYSTORE_MINIO_ACCESSKEY=minio
-      - RP_BINARYSTORE_MINIO_SECRETKEY=minio123
-      - LOGGING_LEVEL_ORG_HIBERNATE_SQL=info
-      - JAVA_OPTS=-Xmx1g -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp  -Dcom.sun.management.jmxremote.rmi.port=12349 -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.local.only=false  -Dcom.sun.management.jmxremote.port=9010 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=0.0.0.0
+      ## Double entry moves test logs from PostgreSQL to Elastic-type engines
+      ## Ref: https://reportportal.io/blog/double-entry-in-5.7.2
+      RP_ELASTICSEARCHLOGMESSAGE_HOST: http://opensearch:9200
```

Comment:
in 24.1 the name of this var was changed to RP_ELASTICSEARCH_HOST

Reply:
also variables RP_ELASTICSEARCH_USERNAME and RP_ELASTICSEARCH_PASSWORD need to add